jqm-all/jqm-model/src/main/java/com/enioka/jqm/jdbc/Db.java

```diff
@@ -506,7 +506,24 @@ public DbConn getConn()
         }
         catch (SQLException e)
         {
-            DbHelper.closeQuietly(cnx); // May have been left open when the pool has given us a failed connection.
+            if (DbConn.testDbUnreachable(e))
```

Comment:
Why the need for a static method here?  Could the analysis actually take place inside the DbConn being created and not expose this method?